README.md

```diff
@@ -10,41 +10,49 @@ https://firecloud.dsde-prod.broadinstitute.org/
 
 We use the [Leiningen](http://leiningen.org/) build tool to manage ClojureScript dependencies.
 
-The code incorporates usage of [react-cljs](https://github.com/dmohs/react-cljs) which is 
+The code incorporates usage of [react-cljs](https://github.com/dmohs/react-cljs) which is
 a ClojureScript wrapper for [React](https://facebook.github.io/react/).
 
 Figwheel replaces the running JavaScript within the page so changes are visible without a browser reload. More information [here](https://github.com/bhauman/lein-figwheel). [This video](https://www.youtube.com/watch?v=j-kj2qwJa_E) gives some insight into the productivity gains available when using this technology (up to about 15:00 is all that is necessary).
 
-## Getting Started
+## Getting started
 
 FireCloud is currently supported within Broad's engineering environment:
 
 https://github.com/broadinstitute/firecloud-develop
 
 Support for running FireCloud outside of the Broad is planned.
 
-## Build Options
+## Starting the development server
 
-Build once:
-```
-./script/build-once.sh
+```bash
+./config/docker-rsync-local-ui.sh
 ```
 
-Watch files and rebuild whenever a file is saved:
-```
-./script/start-auto-build.sh
```

Comment:
I went with the delete. My intuition is that these aren't used often-enough to warrant keeping them current. Does that sound right to you?